tests/test_log_command.rs

```diff
@@ -198,3 +198,30 @@ fn test_log_filtered_by_path() {
     A file2
     "###);
 }
+
+#[test]
+fn test_default_revset() {
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_success(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+
+    std::fs::write(repo_path.join("file1"), "foo\n").unwrap();
+    test_env.jj_cmd_success(&repo_path, &["describe", "-m", "add a file"]);
+
+    // Set configuration to only show the initial commit.
+    test_env.add_config(
+        br#"[ui]
+        default-revset = "0000000000000000000000000000000000000000"
```

Comment:
nit: use `root` here (and s/initial/root/ in the comments above and below) since it's both more readable and more portable (they can vary between commit backends)

Reply:
Done.